contracts/upgradeable_contracts/amb_erc677_to_erc677/ForeignAMBErc677ToErc677.sol

```diff
@@ -57,11 +81,28 @@ contract ForeignAMBErc677ToErc677 is BasicAMBErc677ToErc677 {
         bytes _data
     ) internal {
         if (!lock()) {
+            _setMediatorBalance(mediatorBalance().add(_value));
```

Comment:
should it be reflected in `amb_erc677_to_erc677/callflows.md`?

Reply:
In `amb_erc677_to_erc677/callflows.md` we have a behaviour described for Stake pair of mediators only. In Stake mediators, we do not need fixMediatorBalance functionality, since it operates on our own erc677 token.

Reply:
Thanks